### PR TITLE
Add variable number of alert days

### PR DESCRIPTION
**Pull request by Myoldmopar**
Like this?

### CODE REVIEW

**Comment by lefticus** [comment]
That's a start, except for two things:

 1. My preference is to put the configuration defaults in the configuration.rb file: https://github.com/lefticus/decent_ci/blob/3cacc7905702ecc3d5e67a36a05f11c3d01655cf/lib/configuration.rb#L386-L388
 2. You aren't actually getting the parsed configuration data down to the place you are using it. The pull request notification code does not have access to the full configuration, that needs to be passed in here, like it is for the related variables: https://github.com/lefticus/decent_ci/blob/3cacc7905702ecc3d5e67a36a05f11c3d01655cf/lib/build.rb#L108-L129

**Comment by Myoldmopar** [comment]
Interesting, OK, shows where my knowledge of decent_ci is at.  :wink:

I will refine these changes with your comments.  Thanks!

File: lib/build.rb [hunk 1]
```diff
@@ -106,6 +106,7 @@ def query_pull_requests
       end
 
       aging_pull_requests_notification = true
```
Comment (Myoldmopar) [inline lib/build.rb]: @lefticus I mimicked this initialization for my new variable, but I'm not sure if it was actually needed.

**Comment by Myoldmopar** [comment]
@lefticus I think this may be ready to go in; look forward to your comments or merge.

File: lib/build.rb (hunk 1 above)
Comment (lefticus) [inline lib/build.rb]: Yeah, I agree, this code may be redundant, for both counts.

**Comment by lefticus** [comment]
Are you concerned about testing it first? Although that is kind of hard to do...

**Comment by Myoldmopar** [comment]
Well, how easy would that be?  I could change the aging length to 1 day in my working branch over [here](https://github.com/NREL/EnergyPlus/pull/5094) but can we point to a different decent_ci branch?

**Comment by lefticus** [comment]
Yeah, that's the hard part. We'd have to run a new instance of the CI with the new branch. Or we can just merge it and see what happens :)

I'll try a quick local run and see if anything pops up, then I'll merge it.

**Comment by Myoldmopar** [comment]
Thanks @lefticus; I think it should be good to go.

**Comment by lefticus** [comment]
@Myoldmopar FYI, while testing I discovered that a bad results file could cause the CI to be crashing, so the DailyTasks were not being executed. This is merged now.

**Pull request merged**